src/main/java/ch/njol/skript/conditions/base/PropertyCondition.java

```diff
@@ -189,4 +175,19 @@ public static String toString(Condition condition, PropertyType propertyType, @N
 		}
 	}
 
+	@Override
+	public @NotNull Predicate<T> and(@NotNull Predicate<? super T> other) {
+		throw new UnsupportedOperationException("Combining property conditions is undefined behaviour");
+	}
+
+	@Override
+	public @NotNull Predicate<T> negate() {
+		throw new UnsupportedOperationException("Negating property conditions without setNegated is undefined behaviour");
+	}
+
+	@Override
+	public @NotNull Predicate<T> or(@NotNull Predicate<? super T> other) {
+		throw new UnsupportedOperationException("Combining property conditions is undefined behaviour");
+	}
```

Comment:
We have the Conditional API, so this might not be totally unsupportable.

Reply:
im not sure how this would be possible. using Conditional.compound requires the `other` param to also be a Conditional, which isnt guaranteed, and for using `evaluate...` it requires an event as context, which isnt passed through these methods. am i missing something?

Reply:
Yeah, that probably isn't even necessary. I'm still a little 😬 about implementing Predicate here. I mean, I suppose we could allow these methods without any major issue. Though, should the `test` method consider `isNegated()`?